mmv1/products/firebase/terraform.yaml

```diff
@@ -41,8 +41,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   WebApp: !ruby/object:Overrides::Terraform::ResourceOverride
     import_format: ['{{project}} {{name}}']
     autogen_async: true
-    skip_delete: true #currently only able to delete a webapp through the Firebase Admin console
-    skip_sweeper: true
+    skip_sweeper: true # handwritten sweeper to hard-delete
```

Comment:
is the hand-written sweeper not in this PR?

Reply:
Yes its the file below. `skip_sweeper` means "don't generate a sweeper". Kind of a bad name there.